### PR TITLE
feat(pie-docs): DSW-1899 add the overview section of the notification component

### PR DESCRIPTION
**Pull request by raoufswe**
## Describe your changes (can list changeset entries if preferable)

- adds the overview section of the notification component

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] I have added thorough tests where applicable (unit / component / visual)
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] I have reviewed visual test updates properly before approving
- [x] If a changeset file has been created, I have used the `/snapit` functionality to test my changes in a consuming application

### Reviewer 1 - Fernando
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] If there are visual test updates, I have reviewed them

### Reviewer 2 Leksa
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] If there are visual test updates, I have reviewed them

### CODE REVIEW

**Review by fernandofranca — DISMISSED** [review]
🔥🔥🔥

**Review by leksaBoiko — APPROVED** [review]

File: apps/pie-docs/src/components/notification/overview/overview.md [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+    parent: Notification
+    order: 1
+eleventyComputed:
+    usage: "{% include './usage.json' %}"
+shouldShowContents: true,
+---
+
+## Overview
+
+The purpose of notifications is to promptly inform users about relevant information or changes that require their attention. Notifications can be used to communicate messages like new messages, system updates, completed actions, errors or reminders.Notifications often include options for users to interact with the displayed information, such as dismissing the notification, taking action or accessing related content. They play a crucial role in keeping users informed and engaged with the application or system.
```
Comment (sebastienrocks) [inline apps/pie-docs/src/components/notification/overview/overview.md:13]: Design has a line-break after:

"The purpose of notifications is to promptly inform users about relevant information or changes that require their attention. Notifications can be used to communicate messages like new messages, system updates, completed actions, errors or reminders."

File: apps/pie-docs/src/components/notification/overview/overview.md [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+    ]
+} %}
+
+---
+
+## Variations
+
+### Neutral
+
+Neutral  provides generic messages to users that may not be related to the current action or task they’re performing. Does not require immediate action and can be dismissed on a timer or persist, depending on the content.
```
Comment (sebastienrocks) [inline apps/pie-docs/src/components/notification/overview/overview.md:67]: "Neutral  provides" Should be "Neutral notification provides"

File: apps/pie-docs/src/components/notification/overview/overview.md [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+
+{% contentPageImage {
+    src:"../../../assets/img/components/notification/variation-neutral.svg",
```
Comment (sebastienrocks) [inline apps/pie-docs/src/components/notification/overview/overview.md:70]: Is the image correct? On the designs it has a white background over the pie-notification, not grey

File: apps/pie-docs/src/components/notification/overview/overview.md [hunk 4]
```diff
@@ -0,0 +1,401 @@
...
+
+---
+
+## Narrow
+
+All props are available in narrow screen sizes.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/notification/narrow.svg",
+    alt: "The notification component on narrow screens",
```
Comment (sebastienrocks) [inline apps/pie-docs/src/components/notification/overview/overview.md:287]: On the designs, narrow and compact images are aligned left (although I do think they look better center-aligned, maybe just worth syncing up with Rachel or Molly for a sign off?)

Comment (raoufswe) [reply apps/pie-docs/src/components/notification/overview/overview.md:287]: I brought this up with @RLumleyJET and we agreed to keep them centered. 💯

**Review by sebastienrocks — APPROVED** [review]
Design review done

**Review by leksaBoiko — APPROVED** [review]

**Pull request merged**